### DISCUSSION TITLE
Stretch and Scale Selections

### DISCUSSION THREAD

**Discussion by JWForever5504** [Mojang/minecraft-editor#57] [Feedback]
I have seen world editors (I don't know which ones) that are able to stretch and scale builds (i.e.: Take a 12x12x12 structure and make it a 12x24x12 or 24x24x24 structure, by turning each block into a 1x2x1 or 2x2x2 prism made of that block). 

I think this would be an amazing feature, but it could be made even better with the incredibly unlikely addition of turning one block into a cube with each pixel looking like the pixel on that block. For example, taking a grass block and scaling it up to look like [this image](https://user-images.githubusercontent.com/127066569/224092136-55234da6-bfd7-474e-a3ce-b62d798b7aad.png).

While I see this as extremely unlikely and difficult to code/maintain, I can't deny that it would look very cool, and could lead to some very interesting builds.

**Comment by Landy005** [comment]
Need the sauce on those stretching editors 👀 Fawe has //size but no fawe plugin or additional fawe plugin can stretch builds afaik..

**Reply by JWForever5504** [reply]
I saw [this](https://youtu.be/sJBUohD8d_g?t=673), which describes the first thing that I hope for. Pretty cool.

**Reply by Landy005** [reply]
Oh wow didn't know amulet could do that, thanks!

**Comment by minecraftchloe** [comment]
I don't know enough at this time to confidently assert that this could be an Editor extension, but I'm curious if when we release v0.4 if this is something that could be done -- combining the powers of the Editor API, Creator/Script API, and existing commands! 🧐
